dietpi/dietpi-services

```diff
@@ -68,6 +69,8 @@
 			elif [ -f /etc/init.d/"${aSERVICE_NAME[$i]}" ]; then
 				aSERVICE_AVAILABLE[$i]=1
 
+			elif [ -f /etc/systemd/system/"${aSERVICE_NAME[$i]}".service ]; then
```

Comment:
Yep, wheezy is being phased out (as per official raspbian image). We are not adding additional software for it either, so best to remove the installation option for wheezy.